ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/Utils.java

```diff
@@ -88,18 +89,24 @@ public static void writeOutput(List<String> values, Path outputFile, HiveConf hi
   public static Iterable<String> matchesTbl(Hive db, String dbName, String tblPattern)
       throws HiveException {
     if (tblPattern == null) {
-      return getAllTables(db, dbName);
+      return getAllTables(db, dbName, null);
     } else {
       return db.getTablesByPattern(dbName, tblPattern);
     }
   }
 
-  public static Collection<String> getAllTables(Hive db, String dbName) throws HiveException {
+  public static Iterable<String> matchesTbl(Hive db, String dbName, ReplScope replScope)
```

Comment:
In case of bootstrap, db name inside ReplScope can be Db pattern. But dbName passed here is each DB that matches the pattern. In bootStrapDump() method, we can see the corresponding code. It is base code behaviour and so kept it same.